package.json

```diff
@@ -45,5 +51,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "jest": {
+    "transformIgnorePatterns": [
+      "node_modules/(?!@ngrx|(?!deck.gl)|ng-dynamic)"
+    ]
```

Comment:
There were a couple of libraries which were complaining without this (e.g. `camelize-ts`), I'm assuming we wouldn't have this in the frontend repo, again just adding here for ease.